install/threesdk/cli.py

```diff
@@ -132,36 +132,40 @@ def update(branch="master"):
         # pull all repos in containers
         for item in containers_names:
             name = item.name
-            print("Updating repositories in container ", name, "...")
             container._containers.assert_container(name)
             c = DockerFactory.container_get(name=name)
             if c.mount_code_exists:
                 host_mount = True
             else:
+                print("Updating repos in container ", name, "...")
                 container_executor = c.executor
                 installer.repos_get(pull=True, executor=container_executor, branch=branch)
 
         if host_mount:
+            print("Updating repos on host...")
             installer.repos_get(pull=True, branch=branch)
 
         # restart containers
         for item in containers_names:
             name = item.name
+            c = DockerFactory.container_get(name=name)
             if name == "simulator":
                 simulator.restart(container=True)
             elif name == "3bot":
                 threebot.restart(container=True)
             else:
-                res = c.execute(
-                    "ps -ef | grep /sandbox/var/cmds/threebot_default.py | grep -v grep", die=False, showout=True
+                rc, out, err = c.execute(
+                    "ps -ef | grep /sandbox/var/cmds/threebot_default.py | grep -v grep",
```

Comment:
suggest to replace this with
3BOTDEVEL:3bot:~: pgrep -f /sandbox/var/cmds/threebot_default.py
4971

Reply:
replaced, thanks Jo.